### DISCUSSION TITLE
stabilityDays/minimumReleaseAge to last patch version

### DISCUSSION THREAD

**Discussion by pando85** [renovatebot/renovate#29393] [Request Help]
### What would you like help with?

Other

### How are you running Renovate?

None

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

v40.1.11 on Github

### Please tell us more about your question or problem

I want to wait for X days for the next minor/major version but when I upgrade to that version I would like to install the latest patch.

With the current configuration, I just can configure it to wait for X days and I don't have access to the last version in the same PR.

Current configuration:
```
      matchUpdateTypes: ["major", "minor"],
      minimumReleaseAge: "30 days",
```

With the current configuration, I get this kind of upgrade workflow:
```
1.1.10 -> 1.2.0 -> 1.2.5
```
I would like to avoid the 1.2.0 jump

Desired:
```
1.1.10 -> 1.2.5
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Unfortunately there's no way to achieve this currently

**Reply by pando85** [reply]
Thank you for your response. Confirmation makes me to rest about looking the way to.

Did you think in adding something like that it should I think in trying to do a contribution here? 

In that case how use to be the process for defining the feature requirements.

**Reply by rarkins** [reply]
Yes, I have thought about this idea in the past and think it would be good. Rather than calling it a "release age" it would be something like "delay" or "waiting period", e.g. major waiting period = 1 month, minor waiting period = 1 week. You'd have to be careful about frequently released packages though for the minor one.

**Reply by HonkingGoose** [reply]
I like the waiting period term! If we only support intervals of a day, a name like `waitingPeriodInDays` could work.

Should we rename this concept a third time? 🙃

1. `stabilityDays`
2. `minimumReleaseAge`
3. `waitingPeriodInDays` or just `waitingPeriod`

The rename could go in the upcoming v38 release maybe?

**Reply by HonkingGoose** [reply]
@rarkins do you want an issue from me to:

- Rename `minimumReleaseAge` to `waitingPeriod` or `waitingPeriodInDays`? (Let me know _which_ term you want!)
- Label the issue as `v38` and `breaking change`?

**Reply by rarkins** [reply]
I think these are separate concepts and renaming is not best for now
